src/sentry/static/sentry/app/views/onboarding/configure/index.jsx

```diff
@@ -41,13 +49,36 @@ const Configure = React.createClass({
     this.api.request(`/projects/${orgId}/${projectId}/events/`, {
       method: 'GET',
       success: data => {
-        // this indicates that a real event has been sent to the project (the first one is the sample event)
-        if (data.length > 1) {
-          this.redirectUrl();
+        let {isFirst, hasSent} = this.state;
+
+        // this indicates that a real event has been sent to the project
+        let sentEvent = function() {
+          if (data.length == 1) {
+            let firstError = data[0];
+            return !firstError.message.includes('This is an example');
+          } else {
+            return data.length > 1;
+          }
+        };
+
+        if (isFirst) {
```

Comment:
We discussed this IRL, but it'd be good to move the logic that redirects outside of the success function into a separate function for compartmentalization/readability.